lib/elixometer.ex

```diff
@@ -281,6 +281,9 @@ defmodule Elixometer do
   @doc """
   Updates a counter metric. If the metric doesn't exist, the metric is created
   and the metric is subscribed to the default reporter.
+
+  If the value of the `:reset_seconds` option is greater than zero, the counter will be reset
+  automatically at the specified interval.
   """
   def update_counter(name, delta, opts \\ [reset_seconds: 60]) when is_bitstring(name) do
```

Comment:
Ah, so you want to disallow specifying 0 or less for reset seconds entirely. Fair enough.